packages/desktop-client/src/components/App.tsx

```
@@ -86,14 +98,16 @@ function AppInner() {
         }),
       );
-      send('get-remote-files').then(files => {
-        if (files) {
-          const remoteFile = files.find(f => f.fileId === cloudFileId);
-          if (remoteFile && remoteFile.deleted) {
-            dispatch(closeBudget());
+      await maybeUpdate(() =>
+        send('get-remote-files').then(files => {
+          if (files) {
+            const remoteFile = files.find(f => f.fileId === cloudFileId);
+            if (remoteFile && remoteFile.deleted) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)